stdlib/public/Darwin/ObjectiveC/ObjectiveC.swift

```diff
@@ -27,26 +27,30 @@ import _SwiftObjectiveCOverlayShims
 public struct ObjCBool : ExpressibleByBooleanLiteral {
 #if os(macOS) || (os(iOS) && (arch(i386) || arch(arm)))
   // On OS X and 32-bit iOS, Objective-C's BOOL type is a "signed char".
-  var _value: Int8
+  @usableFromInline var _value: Int8
```

Comment:
This is technically cheating because this wasn't ABI in 5.0 and therefore there wasn't a symbol there, so we have to really cross our fingers that nothing will stay referencing the symbol. That seems pretty likely, though.

Reply:
Ah hm good point >.<

What about at -Onone? Seems like there’s more risk there maybe?

Reply:
I *think* since the struct is fixed-layout it *should* be okay. It's still usable-from-inline and not public so it's not like it can be used in arbitrary ways.